### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Run spark local console_Screen Reader: Jaws is not reading the result that is displayed on the local console window when we type “sc.appName” and press cntrl + enter.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3619]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #MAS1.3.1; #ScreenReader; #Jaws; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28
Screen Reader: JAWS.
MAS Violated: MAS1.3.1

**NOTE:
Same issue is observed in "Send selection to spark console" scenario.**

**Repro Steps:**
1.Turn on Screen Reader.
2.Download IntelliJ application from the URL:  https://www.jetbrains.com/idea/download/#section=windows .
3.Launch IntelliJ IDEA.
4.Tab till Create New Project button and press enter
5.Tab till Azure Spark/HDInsight-->Spark Project Scala
6.Tab till Build Tools-->Maven
7.Tab till Next button and press enter
8.Fill the required details & Tab till finish button and press enter
9.Navigate to Run --> Edit Configurations-->Apache Spark on HDInsight > [Spark on HDInsight] myApp
10.Tab till Locally run tab and press enter.
11.Fill all the required details properly and tab till Ok button and press enter.
12.Tab till Spark local console (Scala) and type “sc.appName” in that window and press cntrl + enter.
13.Verify whether jaws is reading the result that is displayed on the console window or not.

**Actual Result:**
Jaws is not reading the result that is displayed on the local console window when we type “sc.appName” and press cntrl + enter.

**Expected Result:**
Jaws should read the result that is displayed on the local console window when we type “sc.appName” and press cntrl + enter.

**User Impact:**
Screen Reader users will face difficulty in knowing the result that is displayed on the screen if jaws is not reading anything.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_Jaws is not reading the results displayed in console window.pptx](https://github.com/microsoft/azure-tools-for-java/files/3647129/Bug_Jaws.is.not.reading.the.results.displayed.in.console.window.pptx)

**Comment by t-rufang** [comment]
Note: Screen reader is not reading the output of the interactive console.
![image](https://user-images.githubusercontent.com/32627233/65662755-a0784280-e067-11e9-8d21-bf30af2eb6a2.png)

**Comment by t-rufang** [comment]
We are reusing the `org.jetbrains.plugins.scala.console.ScalaLanguageConsole`, which is supported by IntelliJ team and it's out of the scope of our toolkit. Please find the same issue in scala console.
Repro steps:
1. Open the sample project
2. Navigate to a scala file like `LogQuery.scala` file in Project Explorer and open scala file in editor
3. Right click on the coding editing area and select `Run Scala console`
![image](https://user-images.githubusercontent.com/32627233/67269101-9cb1d180-f4e8-11e9-87f3-c3ff8bbeb081.png)
4. Run a command `val x = 5` in the console view and you will find JAWS is not reading the result of the command.
![image](https://user-images.githubusercontent.com/32627233/67269177-cb2fac80-f4e8-11e9-87d2-cec60d53840e.png)

**Comment by t-rufang** [comment]
- Importance of experience: Important. When keyboard user run a script in the interactive console, JAWS is not reading the execution result of the script. Notice that Spark interactive console is one of the core functionalities of our toolkit.
- Quality of experience: Blocking. There is no way for a keyboard user to know the execution result of a script in Spark interactive console.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by v-lasiva** [comment]
As per discussion with SME, Closing the issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749719
